qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
+  Given form "Send CV" with blank fields
+  When I enter valid data in the field "Name"
+  And I enter valid data in the field "Email"
+  And I attach CV
+  And I ticked "I agree to the personal data processing"
+  And I click button "Send"
+  Then my CV has been send
+
+
+  Scenario: exceeding the limit of characters in the field "Name"
+  Given form "Send CV" with blank fields
+  When I enter 71 characters with space delimiter in the field "Name"
+  Then a message appears "The maximum number of characters is 70"
```

Comment:
In this line we usually use the word "should": a message should appear "....."

Reply:
done